js/libs/builder.js

```diff
@@ -703,6 +757,9 @@ var app = new Vue({
         email: data.email,
         type: 'to'
       });
+    },
+    'settings.template': function() {
+      this.editor.setContent(this.settings.description);
```

Comment:
I guess you only need this when it's turned on, right? the first argument in the function should be the value